### PR TITLE
Add pragma on unreachable code from different OSs

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -98,7 +98,7 @@ omit = [
 
 [tool.coverage.report]
 precision = 2
-fail_under = 98.28
+fail_under = 98.50
```
Comment (Kludex) [inline pyproject.toml]:
```suggestion
fail_under = 98.40
```

File: pyproject.toml [hunk 2]
```diff
@@ -98,7 +98,7 @@ omit = [
 
 [tool.coverage.report]
 precision = 2
-fail_under = 98.28
+fail_under = 98.40
```
Comment (Kludex) [inline pyproject.toml]:
```suggestion
fail_under = 98.35
```

**Review by iudeen — APPROVED** [review]
After a long time! Good to be back! :)

**Comment by Kludex** [comment]
Thanks @iudeen 😁

**Pull request merged**